charts/database/templates/database-deployment.yaml

```diff
@@ -45,7 +45,7 @@ spec:
             preStop:
               exec:
                 command:
-                  - gosu
+                  - su-exec
```

Comment:
This is funny, it's a 10kb version of gosu? :D

Reply:
Yes, su-exec can replace Gosu, which is the tool Alpine brings.

Reply:
Very nice!